### PR TITLE
Add preset for leisure=fishing

### PR DESCRIPTION
**Pull request by xmile1**
This PR creates a preset for leisure=fishing
https://github.com/openstreetmap/iD/issues/5469

screen shot 2018-11-09 at 12 35 06 am

### CODE REVIEW

File: data/presets/presets/leisure/fishing.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "aquarium",
```
Comment (xmile1) [inline data/presets/presets/leisure/fishing.json]: I am not certain this is the best icon here, perhaps a new icon needs to be created.

File: data/presets/presets/leisure/fishing.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "maki-aquarium",
```
Comment (xmile1) [inline data/presets/presets/leisure/fishing.json]: I am not certain this is the best icon here, perhaps a new icon needs to be created.

Comment (bhousel) [reply data/presets/presets/leisure/fishing.json]: I agree - you can use [`fas-fish`](https://fontawesome.com/icons/fish?style=solid) for this..

**Comment by bhousel** [comment]
Looks great, thanks @xmile1 👍

**Pull request merged**